cmd/lakefs/cmd/run.go

```diff
@@ -171,26 +171,28 @@ var runCmd = &cobra.Command{
 			idGen = &actions.DecreasingIDGenerator{}
 
 			// init authentication
-			if !cfg.IsAuthTypeAPI() {
+			if authService == nil {
```

Comment:
Just a thought, this could easily go wrong if someone accidentally adds an initialization code for auth service on the wrong line of code. Maybe we should introduce an authServiceType to the configuration (obviously not for this PR)